apps/nextjs-website/src/lib/strapi/fetchFromStrapi.ts

```diff
@@ -26,7 +26,7 @@ export const fetchFromStrapi = <A, O, I>(
           // handle any promise result
           TE.tryCatch(
             () =>
-              fetchFun(`${strapiEndpoint}/api/${path}/?populate=${populate}`, {
+              fetchFun(`${strapiEndpoint}/api/${path}/?${populate}`, {
```

Comment:
The type of [populate](https://github.com/pagopa/developer-portal/pull/615/files#diff-2eba0eebf80badd5106cdee3c81831e67e8bfc5f077c7046d7d06b9ba547c819R12) should change to `string` only, the `*` doesn't work anymore

Reply:
You are totally right, I forgot to remove that. Thanks!